input/includes/resourcereferencesummary.md

```diff
@@ -0,0 +1,32 @@
+|Resource Type|Profile|Supported Target Reference|
+|---|---|---|---|
+|AllergyIngolerance|[AU Core AllergyIntolerance](StructureDefinition-au-core-allergyintolerance.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|
+|Condition|[AU Core Condition](StructureDefinition-au-core-condition.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|
+|DocumentReference|[DocumentReference](https://hl7.org/fhir/R4/documentreference.html)|-|
+|Encounter|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|[AU Core Condition](StructureDefinition-au-core-condition.html), [AU Core Location](StructureDefinition-au-core-location.html), [Observation](http://hl7.org/fhir/R4/observation.html), [AU Core Organization](StructureDefinition-au-core-organization.html), [AU Core Patient](StructureDefinition-au-core-patient.html), [AU Core Practitioner](StructureDefinition-au-core-practitioner.html), [AU Core PractitionerRole](StructureDefinition-au-core-practitionerrole.html), [AU Core Procedure](StructureDefinition-au-core-procedure.html), [AU Base Related Person](http://hl7.org.au/fhir/StructureDefinition/au-relatedperson)|
+|Extension|[AU Core Sex Assigned At Birth](StructureDefinition-au-core-rsg-sexassignedab.html)|-|
```

Comment:
Please remove; not MS and if it was it would be included in this table through a profile that references it.

Reply:
Removed, thanks.